[2016-05-03 13:48:06] production.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [1044] Access denied for user ''@'localhost' to database 'forge'' in C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:47
Stack trace:
#0 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(47): PDO->__construct('mysql:host=loca...', 'forge', '', Array)
#1 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 C:\xampp\htdocs\erp\api\vendor\compiled.php(10673): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 C:\xampp\htdocs\erp\api\vendor\compiled.php(10669): Illuminate\Database\Connectors\ConnectionFactory->createSingleConnection(Array)
#4 C:\xampp\htdocs\erp\api\vendor\compiled.php(10587): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#5 C:\xampp\htdocs\erp\api\vendor\compiled.php(10542): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#6 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Migrations\DatabaseMigrationRepository.php(167): Illuminate\Database\DatabaseManager->connection(NULL)
#7 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Migrations\DatabaseMigrationRepository.php(135): Illuminate\Database\Migrations\DatabaseMigrationRepository->getConnection()
#8 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(371): Illuminate\Database\Migrations\DatabaseMigrationRepository->repositoryExists()
#9 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Console\Migrations\MigrateCommand.php(101): Illuminate\Database\Migrations\Migrator->repositoryExists()
#10 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Console\Migrations\MigrateCommand.php(54): Illuminate\Database\Console\Migrations\MigrateCommand->prepareDatabase()
#11 [internal function]: Illuminate\Database\Console\Migrations\MigrateCommand->fire()
#12 C:\xampp\htdocs\erp\api\vendor\compiled.php(925): call_user_func_array(Array, Array)
#13 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Console\Command.php(115): Illuminate\Container\Container->call(Array)
#14 C:\xampp\htdocs\erp\api\vendor\symfony\console\Symfony\Component\Console\Command\Command.php(253): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Console\Command.php(101): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 C:\xampp\htdocs\erp\api\vendor\symfony\console\Symfony\Component\Console\Application.php(874): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 C:\xampp\htdocs\erp\api\vendor\symfony\console\Symfony\Component\Console\Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 C:\xampp\htdocs\erp\api\vendor\symfony\console\Symfony\Component\Console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(90): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 C:\xampp\htdocs\erp\api\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 {main}  
[2016-05-03 13:48:08] production.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [1044] Access denied for user ''@'localhost' to database 'forge'' in C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:47
Stack trace:
#0 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(47): PDO->__construct('mysql:host=loca...', 'forge', '', Array)
#1 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 C:\xampp\htdocs\erp\api\vendor\compiled.php(10673): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 C:\xampp\htdocs\erp\api\vendor\compiled.php(10669): Illuminate\Database\Connectors\ConnectionFactory->createSingleConnection(Array)
#4 C:\xampp\htdocs\erp\api\vendor\compiled.php(10587): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#5 C:\xampp\htdocs\erp\api\vendor\compiled.php(10542): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#6 C:\xampp\htdocs\erp\api\vendor\compiled.php(10434): Illuminate\Database\DatabaseManager->connection(NULL)
#7 C:\xampp\htdocs\erp\api\vendor\compiled.php(10421): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#8 C:\xampp\htdocs\erp\api\vendor\compiled.php(9921): Illuminate\Database\Eloquent\Model->getConnection()
#9 C:\xampp\htdocs\erp\api\vendor\compiled.php(9898): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#10 C:\xampp\htdocs\erp\api\vendor\compiled.php(9747): Illuminate\Database\Eloquent\Model->newQueryWithoutScopes()
#11 C:\xampp\htdocs\erp\api\vendor\compiled.php(9394): Illuminate\Database\Eloquent\Model->save()
#12 C:\xampp\htdocs\erp\api\database\seeds\DatabaseSeeder.php(35): Illuminate\Database\Eloquent\Model::create(Array)
#13 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Seeder.php(37): RoleTableSeeder->run()
#14 C:\xampp\htdocs\erp\api\database\seeds\DatabaseSeeder.php(19): Illuminate\Database\Seeder->call('RoleTableSeeder')
#15 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Console\SeedCommand.php(57): DatabaseSeeder->run()
#16 [internal function]: Illuminate\Database\Console\SeedCommand->fire()
#17 C:\xampp\htdocs\erp\api\vendor\compiled.php(925): call_user_func_array(Array, Array)
#18 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Console\Command.php(115): Illuminate\Container\Container->call(Array)
#19 C:\xampp\htdocs\erp\api\vendor\symfony\console\Symfony\Component\Console\Command\Command.php(253): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Console\Command.php(101): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 C:\xampp\htdocs\erp\api\vendor\symfony\console\Symfony\Component\Console\Application.php(874): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 C:\xampp\htdocs\erp\api\vendor\symfony\console\Symfony\Component\Console\Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#23 C:\xampp\htdocs\erp\api\vendor\symfony\console\Symfony\Component\Console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#24 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(90): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#25 C:\xampp\htdocs\erp\api\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#26 {main}  
[2016-05-03 14:02:12] local.ERROR: exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'erp.users' doesn't exist' in C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php:288
Stack trace:
#0 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(288): PDO->prepare('select count(*)...')
#1 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(612): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'select count(*)...', Array)
#2 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(576): Illuminate\Database\Connection->runQueryCallback('select count(*)...', Array, Object(Closure))
#3 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(293): Illuminate\Database\Connection->run('select count(*)...', Array, Object(Closure))
#4 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1377): Illuminate\Database\Connection->select('select count(*)...', Array, true)
#5 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1367): Illuminate\Database\Query\Builder->runSelect()
#6 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1354): Illuminate\Database\Query\Builder->getFresh(Array)
#7 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1636): Illuminate\Database\Query\Builder->get(Array)
#8 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1574): Illuminate\Database\Query\Builder->aggregate('count', Array)
#9 [internal function]: Illuminate\Database\Query\Builder->count()
#10 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(933): call_user_func_array(Array, Array)
#11 C:\xampp\htdocs\erp\api\app\Http\Controllers\CommonController.php(17): Illuminate\Database\Eloquent\Builder->__call('count', Array)
#12 C:\xampp\htdocs\erp\api\app\Http\Controllers\CommonController.php(17): Illuminate\Database\Eloquent\Builder->count()
#13 [internal function]: App\Http\Controllers\CommonController->login()
#14 C:\xampp\htdocs\erp\api\vendor\compiled.php(8287): call_user_func_array(Array, Array)
#15 C:\xampp\htdocs\erp\api\vendor\compiled.php(8354): Illuminate\Routing\Controller->callAction('login', Array)
#16 C:\xampp\htdocs\erp\api\vendor\compiled.php(8333): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\CommonController), Object(Illuminate\Routing\Route), 'login')
#17 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#21 C:\xampp\htdocs\erp\api\vendor\compiled.php(8334): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#22 C:\xampp\htdocs\erp\api\vendor\compiled.php(8320): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\CommonController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'login')
#23 C:\xampp\htdocs\erp\api\vendor\compiled.php(7317): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'login')
#24 C:\xampp\htdocs\erp\api\vendor\compiled.php(7288): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\erp\api\vendor\compiled.php(6954): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\erp\api\vendor\compiled.php(6955): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#31 C:\xampp\htdocs\erp\api\vendor\compiled.php(6944): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#32 C:\xampp\htdocs\erp\api\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\erp\api\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#34 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#36 C:\xampp\htdocs\erp\api\app\Http\Middleware\VerifyCsrfToken.php(18): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#38 C:\xampp\htdocs\erp\api\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#40 C:\xampp\htdocs\erp\api\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#41 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 C:\xampp\htdocs\erp\api\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#43 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#44 C:\xampp\htdocs\erp\api\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#45 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 C:\xampp\htdocs\erp\api\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#49 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#50 C:\xampp\htdocs\erp\api\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#51 C:\xampp\htdocs\erp\api\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#52 C:\xampp\htdocs\erp\api\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#53 {main}

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'erp.users' doesn't exist (SQL: select count(*) as aggregate from `users` where `username` = admin and `password` = c4895d7b6724d48e7019e06b74275eaaaccb562738a89abd05b8500570a4b4d2 and `active` = 1)' in C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php:620
Stack trace:
#0 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(576): Illuminate\Database\Connection->runQueryCallback('select count(*)...', Array, Object(Closure))
#1 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Connection.php(293): Illuminate\Database\Connection->run('select count(*)...', Array, Object(Closure))
#2 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1377): Illuminate\Database\Connection->select('select count(*)...', Array, true)
#3 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1367): Illuminate\Database\Query\Builder->runSelect()
#4 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1354): Illuminate\Database\Query\Builder->getFresh(Array)
#5 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1636): Illuminate\Database\Query\Builder->get(Array)
#6 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1574): Illuminate\Database\Query\Builder->aggregate('count', Array)
#7 [internal function]: Illuminate\Database\Query\Builder->count()
#8 C:\xampp\htdocs\erp\api\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(933): call_user_func_array(Array, Array)
#9 C:\xampp\htdocs\erp\api\app\Http\Controllers\CommonController.php(17): Illuminate\Database\Eloquent\Builder->__call('count', Array)
#10 C:\xampp\htdocs\erp\api\app\Http\Controllers\CommonController.php(17): Illuminate\Database\Eloquent\Builder->count()
#11 [internal function]: App\Http\Controllers\CommonController->login()
#12 C:\xampp\htdocs\erp\api\vendor\compiled.php(8287): call_user_func_array(Array, Array)
#13 C:\xampp\htdocs\erp\api\vendor\compiled.php(8354): Illuminate\Routing\Controller->callAction('login', Array)
#14 C:\xampp\htdocs\erp\api\vendor\compiled.php(8333): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\CommonController), Object(Illuminate\Routing\Route), 'login')
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\erp\api\vendor\compiled.php(8334): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 C:\xampp\htdocs\erp\api\vendor\compiled.php(8320): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\CommonController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'login')
#21 C:\xampp\htdocs\erp\api\vendor\compiled.php(7317): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'login')
#22 C:\xampp\htdocs\erp\api\vendor\compiled.php(7288): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\erp\api\vendor\compiled.php(6954): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\erp\api\vendor\compiled.php(6955): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#29 C:\xampp\htdocs\erp\api\vendor\compiled.php(6944): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#30 C:\xampp\htdocs\erp\api\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#31 C:\xampp\htdocs\erp\api\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\erp\api\vendor\compiled.php(8952): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\erp\api\app\Http\Middleware\VerifyCsrfToken.php(18): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 C:\xampp\htdocs\erp\api\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#38 C:\xampp\htdocs\erp\api\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#40 C:\xampp\htdocs\erp\api\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#41 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 C:\xampp\htdocs\erp\api\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#43 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#44 C:\xampp\htdocs\erp\api\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#45 C:\xampp\htdocs\erp\api\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 C:\xampp\htdocs\erp\api\vendor\compiled.php(8935): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#48 C:\xampp\htdocs\erp\api\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#49 C:\xampp\htdocs\erp\api\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#50 C:\xampp\htdocs\erp\api\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#51 {main}  
